templates/package/partials/favorites.html

```diff
@@ -0,0 +1,57 @@
+{% load partials %}
+
+{% partialdef favorite_btn %}
+    <div class="btn btn-default " id="favorite-btn" hx-post="{% url 'add_favorite' package.id %}" hx-target="#favorite" hx-swap="innerHTML">
+        <span class="glyphicon glyphicon-heart-empty"></span>
+        Add to Favorite
+    </div>
+
+{% endpartialdef %}
+
+
+
+{% partialdef unfavorite_btn %}
+    <div class="btn btn-default " id="unfavorite-btn" onclick="openModal()" >
+        <span class="glyphicon glyphicon-heart"></span>
+        Favorited
+    </div>
+    <dialog>
+        <h3>
+            Do you want to unfavorite {{package.title}}?
+        </h3>
+        <div class="action ">
+            <button class="btn btn-default" onclick="closeModal()" id="no">
+                No
+            </button>
+            <button class="btn btn-default" id="yes" hx-post="{% url 'remove_favorite' package.id %}" hx-target="#favorite" hx-swap="innerHTML">
+                Yes
+            </button>
```

Comment:
I am going to swap the order of these. 

```suggestion
            <button class="btn btn-default" id="yes" hx-post="{% url 'remove_favorite' package.id %}" hx-target="#favorite" hx-swap="innerHTML">
                Yes
            </button>
            <button class="btn btn-default" onclick="closeModal()" id="no">
                No
            </button>
```